src/pages/basic_data/institution_admin/AddEmployeePage.tsx

```diff
@@ -117,60 +120,83 @@ export const AddEmployeePage = () => {
         validationSchema={addEmployeeValidationSchema}
         onSubmit={onSubmit}
         validateOnMount>
-        {({ isSubmitting, values, setFieldValue, errors, isValid }: FormikProps<AddEmployeeData>) => (
-          <Form noValidate>
-            <Box
-              sx={{
-                display: 'grid',
-                gridTemplateColumns: { xs: '1fr', lg: '1fr auto 1fr auto 1fr auto 1fr' },
-                gap: '1rem',
-                mt: '2rem',
-              }}>
-              <FindPersonPanel />
-              <Divider orientation="vertical" />
-              <AddAffiliationSection />
-              <Divider orientation="vertical" />
-              <RolesFormSection
-                personHasNin={!values.person.nvi?.verifiedAt.id}
-                roles={values.roles}
-                updateRoles={(newRoles) => {
-                  if (!newRoles.includes(RoleName.PublishingCurator)) {
-                    newRoles = newRoles.filter(
-                      (role) => role !== RoleName.CuratorThesis && role !== RoleName.CuratorThesisEmbargo
-                    );
-                  }
+        {({ submitForm, isSubmitting, values, setFieldValue, errors, isValid }: FormikProps<AddEmployeeData>) => {
+          return (
```

Comment:
Ser ut som det meste av diffen skyldes at det er lagt inn en ny indent her som egentlig ikke trengs
```suggestion
        {({ submitForm, isSubmitting, values, setFieldValue, errors, isValid }: FormikProps<AddEmployeeData>) => (
```